src/main/cpp/subsystems/Arm.cpp

```diff
@@ -5,23 +5,20 @@
 
 
 bool Arm::Init() {
-    // initializing the arm
-    control_state_ = INIT;
-    
     //pulls PID values from the parameters.toml file and initializes the PID controllers
     double arm_kP = RobotParams::GetParam("arm.lift_pid.kP", 0.005);
     double arm_kI = RobotParams::GetParam("arm.lift_pid.kI", 0.0);
     double arm_kD = RobotParams::GetParam("arm.lift_pid.kD", 0.0);
     double arm_kF = RobotParams::GetParam("arm.lift_pid.kF", 0.05);
     arm_pid_ = std::make_shared<frc::PIDController>(arm_kP, arm_kI, arm_kD);
-    arm_pid_->SetTolerance(2, 2);
+    arm_pid_->SetTolerance(2.5, 3.0);
```

Comment:
todo: parameters

Reply:
yeah I think that's getting taken care of in feat/more-parameterization